lib/flowViewport/selection/fvpSelection.h

```diff
@@ -95,6 +113,18 @@ class Selection
     // Maps prim path to data sources to be returned by the vector data
     // source at locator selections.
     std::map<PXR_NS::SdfPath, _PrimSelectionState> _pathToState;
+
+    //We need to keep an order of selected paths to deal with the lead of the selection, the lead is the last of the selected items
+    PXR_NS::SdfPathVector _selectedPaths;
+
+    bool _IsLastSelected(const PXR_NS::SdfPath& primPath)const;
+
+    //Colors used by wireframe selection highlighting
+    PXR_NS::GfVec4f _activeWireframeColor;
+    PXR_NS::GfVec4f _leadWireframeColor;
+    PXR_NS::GfVec4f _dormantWireframeColor;
```

Comment:
Not sure I can make them const since they are initialized with :
Selection::Selection()
{
    Fvp::ColorPreferences::getInstance().getColor(FvpColorPreferencesTokens->wireframeSelection, _leadWireframeColor);
    Fvp::ColorPreferences::getInstance().getColor(FvpColorPreferencesTokens->wireframeSelectionSecondary, _activeWireframeColor);
    Fvp::ColorPreferences::getInstance().getColor(FvpColorPreferencesTokens->polymeshDormant, _dormantWireframeColor);
}

And there is a possibility that they are updated by the user in the UI, but at this time we need to switch to VP2 and back to Hydra to update.

Reply:
Great point.  Since Fvp::ColorPreferences is a Ufe::Subject and is therefore observable, should we enter a JIRA ticket to do this work later, or do you want to do this in the current pull request?  Completely up to you, there are advantages and disadvantages to each approach (current PR is already big, but on the other hand you know pretty much what to do and there is less process to doing it now).